.github/workflows/security.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run GoSec Security Scanner
-        uses: securego/gosec@master
+        uses: securego/gosec@v2.21.0
```

Comment:
I had to do that to fix the pipeline. The latest version produces an unsupported format which results in error **Unable to upload "gosec-results.sarif" as it is not valid SARIF**